config.js

```diff
@@ -459,6 +459,9 @@ var config = {
     // when the toolbar is shown on mouse movements
     // disable1On1Mode: null | false | true,
 
+    // Default remote name to be displayed
+    defaultRemoteDisplayName: 'Fellow Jitster',
```

Comment:
When we add this, it makes sense to start using it and for web and deprecate using the interface config one, but still read it from there as we did for the rest of the deprecations. 
And handle the case when this is missing in the config.js.